### PR TITLE
LINK-1634 | 3/3 Fix scripts, docker build and TestCafe tests

### PR DESCRIPTION
**Pull request by jorilindell**
## Description 
- Use `import.meta.env` instead of `process.env` also in scripts to generate `robots.txt` and `sitemap`
- Fix Docker build
- Move constants which are using import.meta.env to separate file to avoid errors in TestCafe tests

This is one of 3 PRs to migrate Create React App to Vite and Jest to Vitest. The PRs are splitted to following parts:
- Migrate CRA to Vite (https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/247)
- Migrate Jest to Vitest (https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/248)
- **Fix scripts to generate robots.txt and sitemap.xml, fix Docker build and fix TestCafe tests**

## Partially closes
[LINK-1634](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1634)

[LINK-1634]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1634?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ USER default
 RUN yarn && yarn cache clean --force
 
 # Copy all necessary files
-COPY tsconfig.json .eslintignore .eslintrc.json .prettierrc.json .env /app/
+COPY index.html vite.config.ts tsconfig.json .eslintignore .eslintrc.json .prettierrc.json .env* /app/
```
Comment (mikkojamG) [inline Dockerfile:31]: Had some trouble running with Docker, config not being able to find module `vite-plugin-eslint` 🤔

**Review by mikkojamG — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (jorilindell) [reply Dockerfile:31]: For some reason Docker seams to get node_modules from the cache at least in a case that package is moved from devDependencies to dependencies. To be honest, I don't believe that the issue is related to this PR.

I hade to run following commands to get the missing `eslint-plugin-vitest-globals` module, which was missing in my case, to Docker image node_modules:
```
docker system prune
docker compose build
docker compose up
```

Comment (jorilindell) [reply Dockerfile:31]: I think the issue might be related to Docker compose volumes, but this needs little bit more investigation

Comment (jorilindell) [reply Dockerfile:31]: I think the problem is old node_modules which are coming from the volume of the container. To fix this you can run `docker compose down` to remove old container and network or `docker compose down -v` if you also want to remove anonymous volumes.
https://docs.docker.com/engine/reference/commandline/compose_down/

After that `docker compose up` should work normally

Comment (mikkojamG) [reply Dockerfile:31]: Must have been the volumes, now it worked 👍

**Review by mikkojamG — APPROVED** [review]

**Pull request merged**